packages/asc-ui/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { HTMLAttributes } from 'react'
+
+// Some props are omitted since they cannot be overwritten.
+type OmittedProps = 'role' | 'aria-controls' | 'aria-selected' | 'tabIndex'
+
+export interface TabProps
+  extends Omit<HTMLAttributes<HTMLButtonElement>, OmittedProps> {
+  id: string
+  label: string
```

Comment:
You could make this `ReactNode` so that some other elements than `string` can be passed. For example an `<Icon />`.